### ISSUE TITLE
[Bug] Accessibility - Screen reader does not announce editor content

### ISSUE THREAD

**Issue by jquesada2016** [AppFlowy-IO/AppFlowy#7033]
### Bug Description

I am a user who is legally blind. I am using the NVDA screen reader. I noticed that the editor is currently unaccessible while editing content. As you move the text cursor around, nothing is announced by the screen reader, nor are selections. Currently, the only way to have the screen reader announce what's in the editor is to make a selection, copy it, and have the screen reader read aloud the contents of the clipboard. The other way is to mouse over the text, and it will read it aloud, but this is much too slow.

### How to Reproduce

1. Download and install NVDA
2. Open AppFlowy
3. Write some content in the editor
4. Use the arrow keys on your keyboard to move around, and notice nothing is announced by the screen reader
5. Make a text selection, either with the keyboard or mouse, and notice nothing is announced by the screen reader

### Expected Behavior

I expect NVDA to be able to tell when text is edited and the cursor/selection changes state.

### Operating System

Windows 11

### AppFlowy Version(s)

0.7.8

**Comment by KevinCCucumber** [comment]
While not being affilitated with appflowy in any way, this seems to be a flutter issue currently being worked on:
https://github.com/flutter/flutter/issues/103464

**Comment by jquesada2016** [comment]
I just checked out the linked issue. Can we do something different? What we can do is have a hidden alert. Whenever the cursor position changes, or text selection, we can change the content of the hidden alert. This should allow NVDA and other screen readers to pick up the change.

This is definitely not ideal, but it should make at least basic navigation possible.
